lib/mm_network.pm

```diff
@@ -66,16 +66,16 @@ sub configure_dhcp {
     }
     type_string "for MAC in " . join(' ', @mac) . " ; do ";
     type_string "NIC=`grep \$MAC /sys/class/net/*/address |cut -d / -f 5`;";
-    type_string("echo \"STARTMODE='auto'\nBOOTPROTO='dhcp'\n\" > /etc/sysconfig/network/ifcfg-\$NIC;");
-    type_string("done\n");
+    enter_cmd("echo \"STARTMODE='auto'\nBOOTPROTO='dhcp'\" > /etc/sysconfig/network/ifcfg-\$NIC;");
+    enter_cmd("done");
```

Comment:
And the same here: `\n` is followed by other input.

Reply:
done